src/elements/fields/PinInputField.jsx

```diff
@@ -60,7 +92,7 @@ function SingleOtpInput({
         ref={input}
         value={value || ''}
         onChange={onChange}
-        onKeyDown={onKeyDown}
+        // onKeyDown={onKeyDown}
```

Comment:
don't leeave commented out code in production code

Reply:
done